processor/cascadingfilterprocessor/sampling/span_properties_filter_test.go

```diff
@@ -0,0 +1,153 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sampling
+
+import (
+	"math"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.uber.org/zap"
+)
+
+var operationNamePattern = "foo.*"
+var minDurationMicros = int64(500)
```

Comment:
I'd suggest to use the type system: 

```suggestion
var minDuration = 500 * time. Microsecond
```

Reply:
There's a very specific reason I am refering to it as micros. This is the unit for span duration within OT.

Reply:
However, you are right the conversion could happen later. This would make it also much easier on the configuration side